src/components/radio-button/radio-button.js

```diff
@@ -30,14 +31,32 @@ import { tagComponent } from '../../utils/helpers/tags';
  */
 const RadioButton = Input(InputLabel(InputValidation(
 class RadioButton extends React.Component {
+  static propTypes = {
+    /**
+     * A custom class name for the component.
+     *
+     * @property className
+     * @type {String}
+     */
+    className: PropTypes.string,
```

Comment:
It looks like it is possible to whitelist some props that will always be ignored: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md